File: ruby_app/app/models/circle_flavor.rb
class CircleFlavor < ApplicationRecord
	after_initialize :init
  def init
		self.sweetness ||= 0
		self.acidity ||= 0
		self.flowery ||= 0
		self.spicy ||= 0
		self.salty ||= 0
		self.berries ||= 0
		self.chocolate ||= 0
		self.candy ||= 0
		self.body ||= 0
		self.cleaning ||= 0
	end
end

File: ios_app/barista/CircleFlavor.swift
//
//  CircleFlavor.swift
//  barista
//
//  Created by MakingDevs on 8/26/16.
//  Copyright © 2016 MakingDevs. All rights reserved.
//

import Foundation

class CircleFlavor {
    var sweetness: Int?
    var acidity: Int?
    var flowery: Int?
    var spicy: Int?
    var salty: Int?
    var berries: Int?
    var chocolate: Int?
    var candy: Int?
    var body: Int?
    var cleaning: Int?
    
    init(sweetness: Int?, acidity: Int?, flowery: Int?, spicy: Int?, salty: Int?, berries: Int?, chocolate: Int?, candy: Int?, body: Int?, cleaning: Int?) {
        self. sweetness = sweetness
        self.acidity = acidity
        self.flowery = flowery
        self.spicy = spicy
        self.salty = salty
        self.berries = berries
        self.chocolate = chocolate
        self.candy = candy
        self.body = body
        self.cleaning = cleaning
    }
}
